documentation/modules/configuring/proc-config-mirrormaker2-securing-connection.adoc

```diff
@@ -239,8 +427,10 @@ spec:
       topicsPattern: ".*"
       groupsPattern: ".*"
 ----
-<1> The TLS certificates for the source Kafka cluster.
+<1> The TLS certificates for the source Kafka cluster. If they are in a separate namespace, copy the cluster secrets from the namespace of the Kafka cluster.
 <2> The user authentication for accessing the source Kafka cluster using the xref:type-KafkaClientAuthenticationTls-reference[TLS mechanism].
+<1> The TLS certificates for the target Kafka cluster.
+<2> The user authentication for accessing the target Kafka cluster.
 
 . Create or update the `KafkaMirrorMaker2` resource.
```

Comment:
We should probably point the user into which namespace should this be deployed? (the namespace of the target cluster)